share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -68,9 +68,6 @@ installpkg plymouth
 ## extra dracut modules
 installpkg anaconda-dracut dracut-network dracut-config-generic dracut-fips
 
-## import-state.service for switchroot
-installpkg initscripts
```

Comment:
NAK on this line. See https://bugzilla.redhat.com/show_bug.cgi?id=1615332

Reply:
There was no comment from the initscripts maintainer as to why it was needed, and there's also no documentation there as to what broke, it just says "some network configuration use cases are broken" which isn't useful.

Can you merge the arm patch without the other one?